### ISSUE TITLE
Plugin view is unthemed in dark mode

### ISSUE THREAD

**Issue by sawhney17** [rcvd/logseq-quattro-theme#6]
Noticed a bug: in dark mode, the plugin view is green, taking on the default theme and isn't using the custom colors. 

<img width="1440" alt="Screen Shot 2022-02-08 at 10 13 09 AM" src="https://user-images.githubusercontent.com/80150109/152928861-83339624-e936-47cb-9645-833160e30873.png">

**Comment by rcvd** [comment]
Could you please re-check if the issue persists with the latest update?

**Comment by sawhney17** [comment]
<img width="1440" alt="Screen Shot 2022-02-08 at 10 41 55 AM" src="https://user-images.githubusercontent.com/80150109/152932457-9995d51f-f5c5-4788-9eb5-83bfb0422801.png">
Don't think it's fixed yet. I am on the right version 0.015 right?

**Comment by rcvd** [comment]
You are right. I had an interfering custom.css that solved that background issue by setting the default variables to something useful. I had incorporated that into the craft theme for the first time and will re-implement it for all of my other themes now. That way colors should fit better in unthemed areas and plugins which are using the default color variables (which would be a good idea!) will also fit well.

**Comment by sawhney17** [comment]
Got it. Sounds good. Love how the buttons in logseq smartblocks are automatically themed to the correct font and blue highlight in the quattro theme.
